include/create_shadowentry.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file include/create_shadowentry.php
+ * @brief This script creates posts with UID = 0 for a given public post.
+ *
+ * This script is started from mod/item.php to save some time when doing a post.
+ */
+require_once("boot.php");
+require_once("include/threads.php");
+
+function create_shadowentry_run(&$argv, &$argc) {
```

Comment:
If it's not absolutely required, I'd suggest refraining from using the reference operator `&` since it denotes a use outside of the expected behavior.